src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1680,36 +1648,46 @@ void RaftPart::processAppendLogRequest(
         return;
     }
 
-    if (req.get_committed_log_id() > committedLogId_) {
+    LogID lastLogIdCanCommit = std::min(lastLogId_, req.get_committed_log_id());
+    if (lastLogIdCanCommit > committedLogId_) {
         // Commit some logs
         // We can only commit logs from firstId to min(lastLogId_, leader's commit log id),
         // follower can't always commit to leader's commit id because of lack of log
-        LogID lastLogIdCanCommit = std::min(lastLogId_, req.get_committed_log_id());
-        CHECK_LE(committedLogId_ + 1, lastLogIdCanCommit);
-        if (commitLogs(wal_->iterator(committedLogId_ + 1, lastLogIdCanCommit))) {
+        auto code = commitLogs(wal_->iterator(committedLogId_ + 1, lastLogIdCanCommit), false);
```

Comment:
One question, when` wait == false` , we will ignore whether the log commits successfully or not.but why do you need to set the committedLogId_ at line 1671?

Reply:
L1671 only set committedLogId_ in resp, but we didn't update the `committedLogId_`, when we commit successfully, we update the committedLogId_, and response the new one (Line 1661 1662).